src/handler.ts

```diff
@@ -1,9 +1,9 @@
 import { Logger, LogMessage, LogMessageType } from './utils/logger';
 import { Operator } from './operator';
 import { DataLayerDetail, createEventType } from './event';
-import DataLayerTarget from './target';
 import { FanOutOperator } from './operators/fan-out';
 import { Telemetry, telemetryType, errorType } from './utils/telemetry';
+import DataLayerValue from './value';
```

Comment:
Added a new interface called DataLayerValue to abstract what is needed from a source.  It used to be a DataLayerTarget, but not all sources will be data layer targets and need everything.